sympy/polys/tests/test_ring_series.py

```diff
@@ -271,6 +273,17 @@ def test_atan():
         *x**3*y + EX((3*a**2 - 1)/(3*a**6 + 9*a**4 + 9*a**2 + 3))*x**3 + \
         EX(1/(a**2 + 1))*x**2*y - EX(a/(a**4 + 2*a**2 + 1))*x**2 + EX(1/(a**2 \
         + 1))*x + EX(atan(a))
+    # Complex numbers not in ring_series
+    R, x = ring('x', EX)
+    assert rs_atan(x*I,x,3) == EX(I)*x
```

Comment:
This is not a good test because the result is the same as the argument.
```
>>> EX(I)*x == I*x
True
```

Reply:
I have deleted the test